File: Chapter1/python/download-png.py
import urllib.request

# サンプル画像の保存

# urllib.request.urlretrieve("http://ff-theworld.club/wp-content/uploads/2016/01/12-28.jpg", "sample.png")
# ↓行儀の良い書き方
mem = urllib.request.urlopen("http://ff-theworld.club/wp-content/uploads/2016/01/12-28.jpg").read()

with open("sample.png", mode="wb") as f:
    f.write(mem)

print("保存完了しました。")

File: Chapter1/ruby/download-png.rb
require "open-uri"

puts "URLを指定してください(空白の場合FFの画像をDWします)"
print "URL:"; url = gets.chomp

url = "http://ff-theworld.club/wp-content/uploads/2016/01/12-28.jpg" if url.empty?

# サンプル画像のダウンロード
open(url) { |image|
  File.open("sample.jpg","wb") do |file|
    file.puts image.read
  end
}

puts "保存完了しました。"
